bag_verifier/src/test/scala/uk/ac/wellcome/platform/archive/bagverifier/services/BagVerifierTestCases.scala

```diff
@@ -76,21 +76,21 @@ trait BagVerifierTestCases[Verifier <: BagVerifier[
   def withBag[R](
     space: StorageSpace,
     externalIdentifier: ExternalIdentifier,
-    bagBuilder: BagBuilder[BagLocation, BagPrefix, Namespace] = bagBuilder
-  )(testWith: TestWith[(Bucket, BagPrefix), R]): R =
+    bagBuilder: BagBuilder[BagLocation, BagPrefix, Namespace] = bagBuilder,
+    version: BagVersion = BagVersion(randomInt(from = 2, to = 10))
+  )(namespace: Namespace)(testWith: TestWith[(Bucket, BagPrefix), R]): R =
```

Comment:
yeah, although it took me a while to understand why the test that I was expecting to fail was in fact succeeding and that was because the namespace was hidden inside the `withBag` method. Maybe it's not all that bad that we make the namespace more explicit